README.md

```diff
@@ -189,6 +190,66 @@ fluffy.save(function () {
 });
 ```
 
+### Track deletion action ID
+
+```javascript
+var mongoose_delete = require('mongoose-delete');
+
+var PetSchema = new Schema({
+    name: String
+});
+
+PetSchema.plugin(mongoose_delete, { deletedId: true });
+
+var Pet = mongoose.model('Pet', PetSchema);
+
+var fluffy = new Pet({ name: 'Fluffy' });
+
+fluffy.save(function () {
+    // mongodb: { deleted: false, name: 'Fluffy' }
+
+    var actionId = mongoose.Types.ObjectId("61fc42a56b4a6670076b16bf");
+
+    // Pass deletedId as an option
+    fluffy.delete({ deletedId: actionId }, function () {
+        // mongodb: { deleted: true, name: 'Fluffy', deletedId: ObjectId("61fc42a56b4a6670076b16bf")}
+
+        fluffy.restore(function () {
+            // mongodb: { deleted: false, name: 'Fluffy' }
+        });
+    });
+});
```

Comment:
🤔 You'll need to turn this into async/await code for it to make any sense. It'll be a horrible mess of callback code otherwise.